src/content/reference/supported-github-actions.mdx

```diff
@@ -19,9 +19,10 @@ The GitHub Actions for Okteto are available directly from [the GitHub Marketplac
 - [Create Namespace](https://github.com/marketplace/actions/create-namespace)
 - [Delete Namespace](https://github.com/marketplace/actions/delete-namespace)
 - [Deploy Pipeline](https://github.com/marketplace/actions/okteto-pipeline)
-- [Destroy Pipeline](https://github.com/marketplace/actions/destroy-okteto-pipeline)
 - [Deploy Preview](https://github.com/marketplace/actions/deploy-preview-environment)
+- [Destroy Pipeline](https://github.com/marketplace/actions/destroy-okteto-pipeline)
 - [Destroy Preview](https://github.com/marketplace/actions/destroy-preview-environment)
+- [Test](https://github.com/marketplace/actions/okteto-test)
```

Comment:
Should we call it "Okteto Test"?

Reply:
I am ok either way, but we should be consistent. For example, we call it "build" instead of "Okteto Build"

Reply:
Merging to have it in before leaving on PTO, but you can send a follow up PR to update names